tilings/strategies/fusion/disjoint_fusion.py

```diff
@@ -0,0 +1,95 @@
+from typing import Iterator, Optional, Tuple
+
+from comb_spec_searcher import DisjointUnionStrategy, StrategyFactory
+from comb_spec_searcher.exception import StrategyDoesNotApply
+from permuta.misc import DIR_NONE
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import Fusion
+
+
+class DisjointFusionStrategy(DisjointUnionStrategy[Tiling, GriddedPerm]):
+    def __init__(self, row_idx=None, col_idx=None):
+        self.col_idx = col_idx
+        self.row_idx = row_idx
+        super().__init__(
+            ignore_parent=False, inferrable=True, possibly_empty=False, workable=True
+        )
+
+    def fusion_algorithm(self, tiling: Tiling) -> Fusion:
+        return Fusion(tiling, row_idx=self.row_idx, col_idx=self.col_idx, tracked=False)
+
+    def decomposition_function(self, comb_class: Tiling) -> Tuple[Tiling, ...]:
+        algo = self.fusion_algorithm(comb_class)
+        if algo.fusable():
+            fused_tiling = algo.fused_tiling()
+            point_count = sum(algo.min_left_right_points())
```

Comment:
This point counting should be discussed later (made a not in the agenda). Can be merged now and we worry about generalizing to more positive tilings later